src/hooks/useStoneSoupGitHubApp.ts

```diff
@@ -0,0 +1,38 @@
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+
+enum ConsoleDotEnvironments {
+  dev = 'dev',
+  stage = 'stage',
+  qa = 'qa',
+  prod = 'prod',
+}
+
+type StoneSoupGitHubAppDataType = {
+  url: string;
+  name: string;
+};
+
+export const StoneSoupGitHubAppData: { [env: string]: StoneSoupGitHubAppDataType } = {
+  dev: { url: 'https://github.com/apps/appstudio-staging-ci', name: 'appstudio-staging-ci' },
+  stage: {
+    url: 'https://github.com/apps/red-hat-appstudio-ci-cd-staging',
+    name: 'red-hat-appstudio-ci-cd-staging',
+  },
```

Comment:
If I am not wrong  chrome will treat `console.dev.redhat.com` as `dev` environment, in that case we should interchange this dev and stage values. 


```suggestion
  stage: { url: 'https://github.com/apps/appstudio-staging-ci', name: 'appstudio-staging-ci' },
  dev: {
    url: 'https://github.com/apps/red-hat-appstudio-ci-cd-staging',
    name: 'red-hat-appstudio-ci-cd-staging',
  },
```

Reply:
I added a dev value because eventually we might have a dev github app if there's a third cluster called dev. And I am using dev (old staging github app) as a default case in the hook so that it works for local dev as well as any other environment value that we do not know.

Reply:
Stage is actually named `stage` because of the new staging cluster that the github app talks to.